language_formatters_pre_commit_hooks/__init__.py

```diff
@@ -11,12 +10,8 @@ def _get_default_version(tool_name: str) -> str:  # pragma: no cover
     The method is intended to be used only from language_formatters_pre_commit_hooks modules
     """
     try:
-        with open(
-            pkg_resources.resource_filename(
-                "language_formatters_pre_commit_hooks",
-                "{tool_name}.version".format(tool_name=tool_name),
-            )
-        ) as f:
+        path = pkg_resources.resource_filename("language_formatters_pre_commit_hooks", f"{tool_name}.version")
```

Comment:
the usage of f-strings will make the code not compatible with Python 3.5

Given that the friction of using `format(...)` approach is limited I would not drop practical usage of the tool on Python 3.5